release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1053,15 +1053,16 @@ Before this update, SRO did not support connecting to registries in disconnected
 [id="ocp-4-11-RAN-disabling-chronyd"]
 ====  Disable chronyd in the PolicyGenTemplate CR
 
-You must disable `chronyd` if you update to {product-title} 4.11 from earlier versions. You can disable `chronyd` in the `[service]` section under `.spec.profile.data` in the `TunedPerformancePatch.yaml` file, which is included in the group `PolicyGenTemplate` CR. Add the following line in `TunedPerformancePatch.yaml`:
+You must disable `chronyd` if you update to {product-title} 4.11 from earlier versions. To disable `chronyd`, add the following line in the `[service]` section under `.spec.profile.data` of the `TunedPerformancePatch.yaml` file. The `TunedPerformancePatch.yaml` file is included in the `source-crs` directory of the `ztp-site-generate` container image:
```

Comment:
This is referring to the PolicyGenTemplate CR in the group policies (vs source-crs). 
    - fileName: TunedPerformancePatch.yaml
      policyName: "config-policy"
      spec:
        profile:
          - name: performance-patch
            # The 'include' line must match the PerformanceProfile metadata.name above (openshift-node-performance-${metadata.name})
            # And the cmdline_crash CPU set must match the 'isolated' set in the PerformanceProfile above
            data: |
              [main]
              summary=Configuration changes profile inherited from performance created tuned
              include=openshift-node-performance-openshift-node-performance-profile
              [bootloader]
              cmdline_crash=nohz_full=2-23,26-47
              [sysctl]
              kernel.timer_migration=1
              [service]
              service.stalld=start,enable
              service.chronyd=stop,disable